source/examples/xgboost-dask-databricks/notebook.ipynb

```diff
@@ -74,7 +74,7 @@
     "* Create a new multi-node cluster that uses the init script\n",
     "* Once the cluster is running upload this notebook to Databricks and continue running these cells on there\n",
     "\n",
-    "```{docref} /platforms/databricks\n",
+    "```{docref} https://github.com/rapidsai/deployment/blob/v25.08.00/source/platforms/databricks.md#multi-node-dask-cluster\n",
```

Comment:
If the instructions up to now are not necessarily working, do we want to point to them?

Reply:
The alternative is a broken link. Linking to the original source is not perfect but it's better than nothing.